pkg/refactor/service/helper.go

```diff
@@ -283,20 +284,32 @@ func buildEnvVars(withGrpc bool, c *core.Config, sv *core.ServiceFileData) []cor
 			Name:  util.DirektivProxyHTTP,
 			Value: c.KnativeProxyHTTP,
 		})
+		proxyEnvs = append(proxyEnvs, corev1.EnvVar{
+			Name:  strings.ToLower(util.DirektivProxyHTTP),
+			Value: c.KnativeProxyHTTP,
+		})
 	}
 
 	if len(c.KnativeProxyHTTPS) > 0 {
 		proxyEnvs = append(proxyEnvs, corev1.EnvVar{
 			Name:  util.DirektivProxyHTTPS,
 			Value: c.KnativeProxyHTTPS,
 		})
+		proxyEnvs = append(proxyEnvs, corev1.EnvVar{
+			Name:  strings.ToLower(util.DirektivProxyHTTPS),
```

Comment:
@jensg-st , What does it means to lower-case an env var name?

Reply:
Python needs lowercase other things need uppercase.

Reply:
Oh I see.